### PR TITLE
Direct Message Steps 1-4 completed - Sebrianne

### PR DESCRIPTION
**Pull request by sebrianne**
Please let me know if everything looks good, thank you!
Sebrianne

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+//last modified by sebrianne ferguson on 2/23/2019
```
Comment (PandiyanA) [inline src/main/java/com/google/codeu/data/Datastore.java:17]: This is not necessary. github will track these changes (who, what, when). Please remove it.

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 2]
```diff
@@ -42,6 +44,7 @@ public void storeMessage(Message message) {
     messageEntity.setProperty("user", message.getUser());
     messageEntity.setProperty("text", message.getText());
     messageEntity.setProperty("timestamp", message.getTimestamp());
+    messageEntity.setProperty("recipient", message.getRecipient()); //added by Sebrianne
```
Comment (PandiyanA) [inline src/main/java/com/google/codeu/data/Datastore.java:47]: Same as other comment. Please remove it.
Please apply this comment to all the files.

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 3]
```diff
@@ -68,7 +72,11 @@ public void storeMessage(Message message) {
         String text = (String) entity.getProperty("text");
         long timestamp = (long) entity.getProperty("timestamp");
 
-        Message message = new Message(id, user, text, timestamp);
+        //added by sebrianne
```
Comment (PandiyanA) [inline src/main/java/com/google/codeu/data/Datastore.java:75]: same as other comment.

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 4]
```diff
@@ -68,7 +72,11 @@ public void storeMessage(Message message) {
...
+        String recipient = (String) entity.getProperty("recipient");
+        Message message = new Message(id, user, text, timestamp, recipient);
+
+        //Message message = new Message(id, user, text, timestamp);
```
Comment (PandiyanA) [inline src/main/java/com/google/codeu/data/Datastore.java:79]: This line is not useful anymore. Please remove it.

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 5]
```diff
@@ -77,10 +79,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
     String user = userService.getCurrentUser().getEmail();
     String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+    String recipient = request.getParameter("recipient"); // added by sebrianne
 
-    Message message = new Message(user, text);
+    Message message = new Message(user, text, recipient); //sebrianne added recipient
     datastore.storeMessage(message);
 
-    response.sendRedirect("/user-page.html?user=" + user);
+    response.sendRedirect("/user-page.html?user=" + recipient); //changed from + user to + recipient
```
Comment (PandiyanA) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java:87]: Comment is not useful. github is there to track the diff anyway.

**Review by PandiyanA — COMMENTED** [review]
Sebri: Thanks for working on this. Added few minor comments, please address and send it to me for further review.

**Review by HL3rd — APPROVED** [review]
Follow Pandi's recommendations, I'm approving and leave it to Pandi to do the final check and approval.

**Comment by PandiyanA** [comment]
Since you got another branch and pull request. Let's discard this. 
Use the other branch: sebrianne_steps1-4_v2 which got two commits already.

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 1 above)
Comment (PandiyanA) [reply src/main/java/com/google/codeu/data/Datastore.java:17]: done.

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 3 above)
Comment (PandiyanA) [reply src/main/java/com/google/codeu/data/Datastore.java]: your comment is not valid

**Pull request closed without merging**